### PR TITLE
Specify how many CNAME get generated Try #2 

### LINKED ISSUE #4: Generation of CNAMEs for MSTs
Section 5.2.1:
      in accordance with [RFC7022].  [OPEN ISSUE:  How are CNAMEs
      specified for MSTs?  Are they randomly generated for each
      MediaStream?  If so, can two MediaStreams be synced?]

### PR DESCRIPTION
**Pull request by fluffy**
This fixes #4

Replaces PR https://github.com/rtcweb-wg/jsep/pull/127

### CODE REVIEW

**Comment by fluffy** [comment]
this is not ready for review yet - need to deal with comment about different PC

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -405,6 +405,23 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
+          
+	  <t> The RTCCNAME values provide a conical name for the RTP
+	  endpoint that allow other RTP endpoints to understand which
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: canonica;

**Comment by ekr** [comment]
I cleaned up the text here.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -405,6 +405,23 @@ setRemote(ANSWER) |                                   |
...
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
+          
+	  <t> RTP CNAME values provide a canonical name for the RTP
+	  endpoint, allowing other RTP endpoints to determine which
+          RTP streams are using the same clock and thus which clock
+	  sources can be used that to synchronize media playout. </t>
+
+	  <t> Any MediaStreamTracks which have different clock sources
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I think it would be a good idea to include some of the 'why' behind these statements.

**Comment by juberti** [comment]
The structure of this looks good - just think the text should be expanded a bit.

**Comment by juberti** [comment]
This should incorporate the change in #152 - and reference sec.cname there as well.

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -405,6 +405,29 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I think this section should move to a slightly different place - suggest the parent section, right before "Interactions With Forking". Doesn't really seem right under "Session Description Format".

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -405,6 +405,29 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggest we call the section "RTP CNAME Semantics"

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -405,6 +405,29 @@ setRemote(ANSWER) |                                   |
...
+        <section title="CNAME Semantics" anchor="sec.cname">
+          
+	      <t> RTP CNAME values provide a canonical name for the RTP
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: This still seems like it needs some editorial cleanup. Let's try to merge the rest and then whoever has some cycles and finish tweaking this.

**Comment by juberti** [comment]
lgtm

**Pull request merged**